.gitignore

```diff
@@ -17,4 +17,3 @@ hs_err_pid*
 target
```

Comment:
It seems the PR is close to the end :)

Thus, please add an entry in the CHANGES_NEXT_RELEASE about the changes. The following is the suggested one:

```
 - Hardening: software quality improvement based on ISO25010 recommendations 
```

Reply:
Fixed in b6d7148bbc37f5c8a5de01bf50c6c12a6d80d237